frontend/src/ImageList.js

```diff
@@ -20,7 +20,7 @@ const styles = theme => ({
   media: { height: 120 },
   closeButton: { position: 'absolute', top: -10, right: -10 },
   tagIcon: { position: 'absolute', top: -12, left: -14 },
-  activeImage: { background: 'black', padding: 1 }
+  activeImage: { background: 'white', padding: 1 }
```

Comment:
Does this mean that the current image is not highlighted?
In that case, why don't you just remove the whole activeImage style?

Reply:
This prevents getting a black background in transparent images, but it's true that it also removes the black border to highlight the current image. Will fix that

Reply:
Right, maybe the image could be highlighted with an icon or something.
I don't even know if the highlight is necessary, since it's obvious which image is selected.